nursery/unmanaged-call-dotnet.yml

```diff
@@ -0,0 +1,10 @@
+rule:
+  meta:
+    name: unmanaged call dotnet
```

Comment:
```suggestion
    name: unmanaged .NET call
```

Reply:
We may want to make this more generic to open up the rule to other runtimes that may be supported in the future.